demo/Dockerfile.alpine

```diff
@@ -17,7 +16,7 @@
 #   To run tests, use `--build-arg WITH_TESTS=true`
 #
 #   # Note: In this case, I would suggest to map port `5000` to `5000`. The user can always change the port according to their needs.
-#   docker run -it --rm --init --name agent -p5000:5000 \
+#   docker run -it --rm --init --name agent -p5000:5000 -v ... \
```

Comment:
I’d suggest to add an example volume binding both here and in `Dockerfile`, unless you’d like to move whole note to the docs. I suggest to do this in all four dockerfiles.

Reply:
Ok